README.md

```diff
@@ -76,7 +79,7 @@ cmake --build . -j
 
 #### Windows (x64 only)
 _Prerequisites:_
-- You must set up [Python 3](https://docs.python.org/3/using/windows.html#the-full-installer)(if you have not done so already).
+- You must set up [Python 3](https://docs.python.org/3/using/windows.html#the-full-installer) (if you have not done so already).
```

Comment:
_Nope_ 😄 This can actually be removed so long as we instruct everyone to just use the Vulkan SDK, which should be done either here or by the time we move to the Wiki for a full setup guide